inc/widget/latest_news.php

```diff
@@ -1,13 +1,13 @@
 <?php
 namespace W3TC;
 
-if ( !defined( 'W3TC' ) )
+if ( ! defined( 'W3TC' ) ) {
 	die();
-
+}
 ?>
-<p class="widget-loading hide-if-no-js {nonce: '<?php echo wp_create_nonce( 'w3tc' ); ?>'}">
-	<?php echo __( 'Loading&#8230;' ) ?>
+<p class="widget-loading hide-if-no-js {nonce: '<?php echo esc_attr( wp_create_nonce( 'w3tc' ) ); ?>'}">
+	<?php echo esc_html( __( 'Loading&#8230;' ) ); ?>
 </p>
 <p class="hide-if-js">
-	<?php echo __( 'This widget requires JavaScript.' ) ?>
+	<?php echo esc_html( __( 'This widget requires JavaScript.' ) ); ?>
```

Comment:
Use `esc_html__()` instead of `esc_html( __() )`.

Reply:
Switched to esc_html__